### ISSUE TITLE
DH shadowmap support causes direct light missing when rotating camera to specific angle

### ISSUE THREAD

**Issue by Skvorcl** [X0nk/Bliss-Shader#297]
I wanted to use the DH shadowmap support function to prevent weird Tyndall light showing through the mountain,but it seems enable this function causes direct light missing when I move my camera angle.

https://github.com/user-attachments/assets/019ed2db-6f29-410b-9a75-1264266dbf42

**Comment by Skvorcl** [comment]
Minecraft 1.20.1 Forge
Embeddium 0.3.31
Oculus 1.7.0
Main Branch of Bliss #482
After disabled this function,that situation never showed up.

**Comment by X0nk** [comment]
yeah its a bit iffy, i will have to go back to turning shadow distortion off

**Comment by augustsaintfreytag** [comment]
Is there an update on this? Bliss would really be perfect if it could take the DH geometry into account for shadow rendering. Having the sun shine through mountains really breaks immersion.

**Comment by X0nk** [comment]
sadly DH shadowmap is broken on iris/DH side indefinitely, please give up on it as i have
